packages/block-editor/src/store/defaults.js

```diff
@@ -14,6 +14,7 @@ export const PREFERENCES_DEFAULTS = {
  * @property {boolean} alignWide Enable/Disable Wide/Full Alignments
  * @property {Array} availableLegacyWidgets Array of objects representing the legacy widgets available.
  * @property {boolean} imageEditing Image Editing settings set to false to disable.
+ * @property {string} [imageEditingCrossOrigin] Optional value to add to the image editing img el's `crossOrigin` attribute
```

Comment:
Do we need to specify that this is specifically for image editing? I wonder if it would be better to set this more generally, like if this cross origin value would be used anywhere other than image editing... 🤔 

Other than that, this looks pretty great. I like that we are using settings for it, so that it's easy to control from the back end, and even the front end if we needed.

Reply:
Yes, we should definitely go as far as `imageCrossOrigin`. I've been wondering how specific or generic to make this. Conceptually, I'd expect this to mean "will media be served from a CDN?" and it's something that the plugin or Core should be able to intelligently set.

Reply:
Sounds good, that makes sense. I'll change the name 👍🏻